Gemfile

```diff
@@ -7,6 +7,8 @@ gem "config"
 
 gem "sassc-rails"
 
+gem "bcrypt", "3.1.13"
```

Comment:
Gem này có tác dụng gì em ơi

Reply:
mã hóa password ạ

Reply:
Mỗi mình mã hóa password thôi à em

Reply:
và nó được sử dụng trong xác minh đăng nhập, nó override lại toán tử == tức là hash password người dùng nhập vào rồi so sánh với chuỗi hash mk lưu trong db